portfolio/src/main/webapp/script.js

```diff
@@ -27,13 +27,38 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+/*
 async function getRandomNameUsingAsync() {
   const response = await fetch('/data');
   const name = await response.text();
  console.log('Adding name to dom: ' + name);
   document.getElementsByTagName('body').innerText = name; 
 }
+*/
```

Comment:
Don't leave commented functions as the final part of code.

Reply:
Solved